python/test/test_tile.py

```diff
@@ -106,11 +106,16 @@ def test_find_tile_multiple(self):
         self.assertEqual(result.length, 1)
 
     def test_find_tile_leaf(self):
-        entries = [Entry(100, 1, 1, 0)]
+        entries = [Entry(100, 1, 1, 151)]
         result = find_tile(entries, 150)
         self.assertEqual(result.offset, 1)
         self.assertEqual(result.length, 1)
 
+    def test_find_tile_missing_between(self):
+        entries = [Entry(3, 3, 1, 0),Entry(5, 5, 1, 0)]
+        result = find_tile(entries, 4)
```

Comment:
`find_tile(entries,4)` should return the first Entry(TileID=3) because `Entry(3,3,1,0)` is a leaf and might contain any tile data in the range `[3,5)` - perhaps we should rename it to `find_relevant_entry`?